shell/components/ResourceList/index.vue

```diff
@@ -172,7 +172,7 @@ export default {
      * This covers case 1
      */
     pagination(neu, old) {
-      if (neu && !this.componentWillFetch && this.paginationEqual(neu, old)) {
+      if (neu && !this.componentWillFetch && !this.paginationEqual(neu, old)) {
```

Comment:
symatic fix. paginationEqual was incorrectly returning the opposite so the opposite was checked here. paginationEqual now returns correctly, so this was updated